src/hooks/useErc20.ts

```diff
@@ -4,18 +4,24 @@ import { useAccount, useBalance } from 'wagmi'
 export const useErc20 = () => {
   const { address } = useAccount()
 
+  const shouldFetchBalance = !!address
+
   const { data: erc20Balance, refetch: refetchErc20Balance } = useBalance({
     address: address,
     token: config.iqAddress as `0x${string}`,
+    enabled: shouldFetchBalance,
```

Comment:
`enabled` is already set to false on undefined address
![image](https://github.com/EveripediaNetwork/iq-ui/assets/58448956/956a1391-038f-4728-a6f0-6da6dd2ff232)

Reply:
Enabled prop is to prevent the hook from continuously running when the address is undefined. If it is not specified the hook will run continuously and throw error when the address is undefined